tests/test_monitor.py

```diff
@@ -0,0 +1,73 @@
+from nose.tools import (
+    eq_,
+    set_trace,
+)
+
+from . import DatabaseTest
+
+from core.model import Subject
+
+from monitor import (
+    FASTNameAssignmentMonitor,
+)
+
+
+class TestFASTNameAssignmentMonitor(DatabaseTest):
+
+    def test_process_item(self):
+
+        # Create some Subjects -- some of them have names and some
+        # don't.
+        fast1 = self._subject(Subject.FAST, "fast1")
+        fast2 = self._subject(Subject.FAST, "fast2")
+        fast2.name = "Existing FAST name."
+
+        lcsh1 = self._subject(Subject.LCSH, "lcsh1")
+        lcsh2 = self._subject(Subject.LCSH, "lcsh2")
+        lcsh2.name = "Existing LCSH name."
+
+        # This Subject has a name but no identifier -- this probably
+        # shouldn't happen.
+        missing_identifier = self._subject(Subject.LCSH, "")
+        missing_identifier.identifier = None
+        missing_identifier.name = None
```

Comment:
The comment says this has a name but here it's set to None. Unless if name is something else?

Reply:
The comment was left over from a prior version of this test that didn't test as much. I fixed the comment.